### PR TITLE
[HOPSWORKS-1849]  fix dataset permissions

### PR DESCRIPTION
**Pull request by ErmiasG**

### CODE REVIEW

**Comment by ErmiasG** [comment]
logicalclocks/hops-testing-ee/pull/130

File: docs/user_guide/hopsworks/dataSetShare.rst [hunk 1]
```diff
@@ -1,91 +1,133 @@
...
+.. figure:: ../../imgs/datasets/share.png
+   :alt: Select a project to share the Data Set with
+   :target: `share.png`_
+   :scale: 25%
+   :align: center
+   :figclass: align-center
+ 
+   Select a project to share the Data Set with
+ 
+When shareing a data set you can also choose share permissions. Available permissions are shown in the image below.
```
Comment (tkakantousis) [inline docs/user_guide/hopsworks/dataSetShare.rst:38]:
```suggestion
When sharing a Data Set you can also choose share permissions. Available permissions are shown in the image below.
```

File: docs/user_guide/hopsworks/dataSetShare.rst [hunk 2]
```diff
@@ -1,91 +1,133 @@
...
 select the project to unshare from. Users of the "shared-with" project, need to right-click on the shared Data Set
 and then click ``Remove DataSet``.
-
-
+ 
+Changing share permission
+-------------------------
+ 
+To change permission of a shared Data Set `data owners` can right-click on the Data Set and click ``Share permission``. This will open a popup
+dialog with a table listing all projects the dataset is shared with.
```
Comment (tkakantousis) [inline docs/user_guide/hopsworks/dataSetShare.rst:80]:
```suggestion
dialog with a table listing all projects the Data Set is shared with.
```

File: docs/user_guide/hopsworks/dataSetShare.rst [hunk 3]
```diff
@@ -1,91 +1,133 @@
...
-levels that give higher privileges to projects that can access this Data Set. All access-levels are:
-
-- Owner only: Sets default permissions setting of a Data Set. Only Data Owners will be able to upload/remove files via the Data Sets Browser and only the data set owner via Jobs and Jupyter notebooks.
-
-- Group writable and sticky-bit set: This enables Data Scientists of the parent project and members of projects the Data Set has been shared with to create directories upload files in the Data Set but are allowed to only remove files/dirs they own, via the Data Sets browser.
-
-- Group writable: This is the least strict setting. It allows both Data Owners and Data Scientists to upload/remove files either via the Data Set Browser or via Jobs/Notebooks.
-
+ 
+By default, all members of the project owning the Data Set are allowed only ``read-only`` access to all files and directories of the Data Set. There are two more access
```
Comment (tkakantousis) [inline docs/user_guide/hopsworks/dataSetShare.rst:103]:
```suggestion
By default, all members of the project the Data Set was created in, are allowed only ``read-only`` access to all files and directories of the Data Set. There are two more access
```

**Pull request merged**